wolfssl/ssl.h

```diff
@@ -4436,6 +4436,10 @@ WOLFSSL_LOCAL char* wolfSSL_get_ocsp_url(WOLFSSL* ssl);
 /* Not an OpenSSL API. */
 WOLFSSL_API int wolfSSL_set_ocsp_url(WOLFSSL* ssl, char* url);
 #endif
+#if defined(OPENSSL_EXTRA) || defined(OPENSSL_EXTRA_X509_SMALL)
+WOLFSSL_LOCAL int oid2nid(word32 oid, int grp);
```

Comment:
These look like internal APIs.
Can we put them in internal.h?

Reply:
Moved to `internal.h`. Good suggestion.